### ISSUE TITLE
error accuracy

### ISSUE THREAD

**Issue by TangKun1234** [pytorch/vision#473]
I want to reproduce the accuracy result with AlexNet model, but the accuracy very low, why?  The model really pretrained?

**Comment by utkuozbulak** [comment]
Hey, are you trying to get predictions based directly on the resized images? I ask this because I was as surprised as you are due to low accuracy.

 If so, you are doing it wrong. AlexNet test phase is a bit different than other models. Quoting from the paper:

> At test time, the network makes a prediction by extracting five 224 × 224 patches (the four corner patches and the center patch) as well as their horizontal reflections (hence ten patches in all), and averaging the predictions made by the network’s softmax layer on the ten patches

There is also an additional step (right after above quote in the paper) that includes adding certain values obtained with eigenvectors/values to image pixels which they reported increasing test accuracy by approximately 1%.

You should achieve similar accuracy as the original work when you include above steps. Hope this helps.

**Comment by fmassa** [comment]
Another thing, did you do the right preprocessing of the image, and also set the model to `.eval()` mode?

Closing as this is probably an user error
